archive/b/bash/quicksort.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
```

Comment:
I'm getting bizarre errors here when I run this in an online editor:

```
jdoodle.sh: line 2: $'\r': command not found
jdoodle.sh: line 3: syntax error near unexpected token `$'{\r''
jdoodle.sh: line 3: `qsort_iter() {
'
```

Reply:
Could you please tell me at which online editor you've tried this out?

When I am using the script locally, everything works fine:
```
$ ./quicksort.sh 4 2 3 4 1 1 2 3 4
1 1 2 2 3 3 4 4 4
```

Reply:
I tried JDoodle and a couple others.